recipes/flecs/all/conandata.yml

```
@@ -1,4 +1,7 @@
 sources:
+  "3.1.4":
+    url: "https://github.com/SanderMertens/flecs/archive/v3.1.4.tar.gz"
```

```suggestion
    url: "https://github.com/SanderMertens/flecs/archive/refs/tags/v3.1.4.tar.gz"
```

prefer full ref please